### PR TITLE
MVC - Controller

### PR DESCRIPTION
**Pull request by Casanova569**
Summary:
Objetive: The trello card is considered approved, if you can access the controller path and see the phrase that returns (message).

Trello Card:
Name: MVC - Controllers
Link: https://trello.com/c/KzY22vCG/2-mvc-controllers

### CODE REVIEW

**Comment by mikolaslorant** [comment]
I suggest reading on Styling pull request descriptions for exmaple:
Adding ## to mark titles
https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet

File: src/Controllers/HelloWorld.cs [hunk 1]
```diff
@@ -0,0 +1,27 @@
+using Microsoft.AspNetCore.Mvc;
+using System.Text.Encodings.Web;
+
+namespace src.Controllers
+{
+    public class HelloWorldController : Controller
+    {
+        //
```
Comment (mikolaslorant) [inline src/Controllers/HelloWorld.cs]: remove comments

File: src/Controllers/HelloWorld.cs [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+    // 
+    // GET: /HelloWorld/Welcome/ 
+
+    public string Welcome()
+    {
+        return "This is the Welcome action method...Hello World!!! (Welcome)";
+    }
+    }
+}
```
Comment (mikolaslorant) [inline src/Controllers/HelloWorld.cs]: Add blank line at end of file (EOF)

File: src/Startup.cs [hunk 3]
```diff
@@ -28,15 +26,29 @@ public Startup(IConfiguration configuration)
...
             services.Configure<CookiePolicyOptions>(options =>
             {
                 // This lambda determines whether user consent for non-essential cookies is needed for a given request.
                 options.CheckConsentNeeded = context => true;
                 options.MinimumSameSitePolicy = SameSiteMode.None;
             });
 
-            services.AddMvc().SetCompatibilityVersion(CompatibilityVersion.Version_2_2);
-            
+            //services.AddMvc().SetCompatibilityVersion(CompatibilityVersion.Version_2_2);
```
Comment (mikolaslorant) [inline src/Startup.cs]: remove comments

File: src/src.csproj [hunk 4]
```diff
@@ -1,19 +1,14 @@
...
-
-</Project>
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
+    <AspNetCoreHostingModel>InProcess</AspNetCoreHostingModel>
+  </PropertyGroup>
+  <ItemGroup>
+
```
Comment (mikolaslorant) [inline src/src.csproj]: remove blank lines

**Review by mikolaslorant — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 5]
```diff
@@ -1,2 +1,3 @@
 bin
-obj
\ No newline at end of file
+obj
+.vscode
```
Comment (truccomarcos) [inline .gitignore:3]: The .vscode folder is still on the branch even though we added it to the `.gitignore` file. Try using the method on [here](http://www.codeblocq.com/2016/01/Untrack-files-already-added-to-git-repository-based-on-gitignore) to remove them (watch out on Step 2, there's no need to remove all files, only the `.vscode` folder)

File: src/src.csproj [hunk 6]
```diff
@@ -1,19 +1,14 @@
...
+    <PackageReference Include="Microsoft.AspNetCore.App"/>
+    <PackageReference Include="Microsoft.AspNetCore.Razor.Design" Version="2.2.0" PrivateAssets="All"/>
+    <PackageReference Include="Npgsql.EntityFrameworkCore.PostgreSQL" Version="2.2.0"/>
```
Comment (truccomarcos) [inline src/src.csproj]: This package will not be used on this feature (it's just necessary for the migrations). Make sure you remove it from this branch

File: src/Controllers/HelloWorld.cs [hunk 7]
```diff
@@ -0,0 +1,27 @@
+using Microsoft.AspNetCore.Mvc;
+using System.Text.Encodings.Web;
+
+namespace src.Controllers
+{
+    public class HelloWorldController : Controller
+    {
+    // GET: /HelloWorld/
```
Comment (truccomarcos) [inline src/Controllers/HelloWorld.cs]: Remove this comments

**Review by truccomarcos — CHANGES_REQUESTED** [review]

File: src/Startup.cs [hunk 8]
```diff
@@ -28,13 +26,15 @@ public Startup(IConfiguration configuration)
         // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
+
```
Comment (truccomarcos) [inline src/Startup.cs:24]: Same as mentioned above.

File: src/Controllers/HelloWorld.cs [hunk 9]
```diff
@@ -0,0 +1,19 @@
+using Microsoft.AspNetCore.Mvc;
+using System.Text.Encodings.Web;
+
+namespace src.Controllers
+{
+    public class HelloWorldController : Controller
+    {
+
```
Comment (truccomarcos) [inline src/Controllers/HelloWorld.cs:12]: As a convention, we try to remove empty lines after a `{`. Also, watch out the indentation for the methods.

File: src/Startup.cs [hunk 10]
```diff
@@ -9,10 +9,8 @@
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
-using MvcMovie.Repositories.Database;
 using Microsoft.EntityFrameworkCore;
-using MvcMovie.Models;
-
+using System.Globalization;
```
Comment (truccomarcos) [inline src/Startup.cs]: We have a convention for how we sort packages. We sort them alphabetically, but we put System packages first. In this case, you should have 
```csharp
using System;
using System.Collections.Generic;
using System.Globalization;
using System.Linq;
using System.Threading.Tasks;
using Microsoft.AspNetCore.Builder;
using Microsoft.AspNetCore.Hosting;
using Microsoft.AspNetCore.Http;
using Microsoft.AspNetCore.HttpsPolicy;
using Microsoft.AspNetCore.Mvc;
using Microsoft.EntityFrameworkCore;
using Microsoft.Extensions.Configuration;
using Microsoft.Extensions.DependencyInjection;
using MvcMovie.Models;
using MvcMovie.Repositories.Database;
```

Comment (truccomarcos) [reply src/Startup.cs]: Also, if you select all the usings and click on the 💡, vscode might prompt you to remove unnecesary usings (EFCore might be taken away on this PR)

**Review by truccomarcos — CHANGES_REQUESTED** [review]

File: src/Startup.cs [hunk 11]
```diff
@@ -1,18 +1,15 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Threading.Tasks;
+﻿// src/Repositories/Database/Startup.cs
```
Comment (truccomarcos) [inline src/Startup.cs]: Remove this comment

File: src/Startup.cs [hunk 12]
```diff
@@ -1,18 +1,15 @@
...
-using Microsoft.AspNetCore.HttpsPolicy;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
-using MvcMovie.Repositories.Database;
 using Microsoft.EntityFrameworkCore;
+using MvcMovie.Repositories.DataBase;
 using MvcMovie.Models;
-
+using src.Models;
```
Comment (truccomarcos) [inline src/Startup.cs]: Why is this using necessary?

**Review by truccomarcos — APPROVED** [review]

**Pull request merged**